transcom/mymove
# B 18342 correct date formats for sit related items

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A875404)

## Summary

Branch name is incorrect - this is for 18343

For SIT related functionality, we are replacing the use of the `formatDate` function from `shared/dates` with `formatDateWithUTC`. This should make the dates match the database values, which are in UTC.

Areas touched were:
- Service Item Details
- Shipment Details
- Service Counseling Move Details (test)
- Shipment Info
- SIT calculations

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the office app 
2. Login as a services counselor
3. Verify the move details page contains correct requested, actual, and scheduled dates that match the database values.

1. Access the office app
2. Login as a TOO
3. View the service items on the move task order page
4. Verify the 'First available delivery date' and 'Customer contact attempt' match the database values

1. Create a move, shipment, and SIT service items
2. Verify the dates match the database and any service item calculations are correct.

### Frontend
